modules/nextflow/src/main/groovy/nextflow/cli/CmdClone.groovy

```diff
@@ -68,9 +72,9 @@ class CmdClone extends CmdBase implements HubOptions {
         }
 
         manager.checkValidRemoteRepo()
-        print "Cloning ${manager.project}${revision ? ':'+revision:''} ..."
-        manager.clone(target, revision, deep)
+        print "Cloning ${manager.project}${revisionSuffix} ..."
```

Comment:
Considering the `AssetManager` knows the `revision` it's worth replacing `${manager.project}${revisionSuffix}` with a small helper method like `manager.getFullQualifiedProjectName()` which reports the project plus the revision

Reply:
added and used method `getProjectWithRevision()`